Changing the hypothesis strategies to generate different values in the tensors

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#28325 Changing the hypothesis strategies to generate different values in the tensors**

Differential Revision: [D18021796](https://our.internmc.facebook.com/intern/diff/D18021796)

Commit:
Changing the hypothesis strategies to generate different values in the tensors

Commit:
Update on "Changing the hypothesis strategies to generate different values in the tensors"


Differential Revision: [D18021796](https://our.internmc.facebook.com/intern/diff/D18021796)

raghuramank100 commented:
It looks like some of the tests are failing with NaNs, we need to fix that first.

Commit:
Update on "Changing the hypothesis strategies to generate different values in the tensors"


Differential Revision: [D18021796](https://our.internmc.facebook.com/intern/diff/D18021796)

Commit:
Update on "Changing the hypothesis strategies to generate different values in the tensors"


Differential Revision: [D18021796](https://our.internmc.facebook.com/intern/diff/D18021796)

Commit:
Update on "Changing the hypothesis strategies to generate different values in the tensors"


Differential Revision: [D18021796](https://our.internmc.facebook.com/intern/diff/D18021796)

z-a-f commented on test/common_utils.py:
```diff
@@ -462,7 +460,7 @@ def __exit__(self, exec_type, exec_value, traceback):
                 database=None,
                 max_examples=100,
-                verbosity=hypothesis.Verbosity.quiet))
+                verbosity=hypothesis.Verbosity.normal))
```
This is already landed by @ezyang

Commit:
Update on "Changing the hypothesis strategies to generate different values in the tensors"


Differential Revision: [D18021796](https://our.internmc.facebook.com/intern/diff/D18021796)
